CMakeLists.txt

```diff
@@ -205,6 +210,47 @@ else()
 	MESSAGE("-- Can't find OpenCV. Please specify OpenCV directory using OpenCV_DIR variable")
 endif()
 
+if(OpenMVS_ENABLE_TBB)
+	FIND_PACKAGE(TBB REQUIRED)
+	if(TBB_FOUND)
+		#These files must be included *after* CGAL is loaded, which occurs during the libs/MVS script.
+		LIST(APPEND OpenMVS_EXTRA_LIBS CGAL::TBB_support)
+		MESSAGE(STATUS "TBB ${TBB_VERSION} found.")
+	else()
+		MESSAGE("-- Can't find TBB. Continuing without it.'")
+	endif()
+endif()
+
+FIND_PACKAGE(CGAL REQUIRED)
+if(CGAL_FOUND)
+	include(CGAL_TBB_support)
+	if (EXISTS "${CMAKE_SOURCE_DIR}/cgal-5.5.2")
+		set(CGAL_DIR "${CMAKE_SOURCE_DIR}/cgal-5.5.2")
+		#include_directories(${CGAL_INCLUDE_DIRS})
+		message("==== Success! CGAL_INCLUDE_DIRS=${CGAL_INCLUDE_DIRS}")
+	else()
+		message(STATUS "Warning: Local copy of CGAL not found. Using vcpkg version.")
+	endif()
+	if (TBB_FOUND)
+		include (CGAL_TBB_support)
+		add_compile_definitions(CGAL_LINKED_WITH_TBB)
+	endif()
+	
+	LIST(APPEND OpenMVS_EXTRA_LIBS "CGAL::CGAL;CGAL::TBB_support")
+endif()
+
+set(VCG_ROOT "${CMAKE_SOURCE_DIR}/vcglib")
```

Comment:
Looks like VCG_DIR should be defined as CACHE var. Then FIND_PACKAGE(VCG REQUIRED)